docs/samples/Microsoft.ML.Samples/Dynamic/KeyToValue_Term.cs

```diff
@@ -0,0 +1,111 @@
+            string customizedColumnName = "CustomizedKeys";
+            var customized_pipeline = new WordTokenizeEstimator(ml, "ReviewReverse", "ReviewReverse")
+                .Append(new TermEstimator(ml, "ReviewReverse", customizedColumnName, maxNumTerms: 10, sort:TermTransform.SortOrder.Value));
+
+            // The transformed data.
+            var transformedData_default = default_pipeline.Fit(trainData).Transform(trainData);
+            var transformedData_customized = customized_pipeline.Fit(trainData).Transform(trainData);
+
+            // Small helper to print the text inside the columns, in the console. 
+            Action<string, IEnumerable<VBuffer<uint>>> printHelper = (columnName, column) =>
+            {
+                Console.WriteLine($"{columnName} column obtained post-transformation.");
+                foreach (var row in column)
+                {
+                    foreach (var value in row.Values)
+                        Console.Write($"{value} ");
+                    Console.WriteLine("");
+                }
+
+                Console.WriteLine("===================================================");
+            };
+
+            // Preview of the DefaultKeys column obtained after processing the input.
+            var defaultColumn = transformedData_default.GetColumn<VBuffer<uint>>(ml, defaultColumnName);
+            printHelper(defaultColumnName, defaultColumn);
+
+            // DefaultKeys column obtained post-transformation.
+            // 1 2 3 4
+            // 5 2 3 1
+            // 6 7 3 1
+            // 8 9 3 1
+
+            // Previewing the CustomizedKeys column obtained after processing the input.
+            var customizedColumn = transformedData_customized.GetColumn<VBuffer<uint>>(ml, customizedColumnName);
+            printHelper(customizedColumnName, customizedColumn);
+
+            // CustomizedKeys
+            // 6 4 9 3
+            // 7 4 9 6
+            // 1 5 9 6
+            // 2 8 9 6
+
+            // Retrieve the original values, by appending the KeyToValue etimator to the existing pipelines
+            // to convert the keys back to the strings.
+            var pipeline = default_pipeline.Append(new KeyToValueEstimator(ml, defaultColumnName));
+            transformedData_default = pipeline.Fit(trainData).Transform(trainData);
+
+            // Preview of the DefaultColumnName column obtained.
+            var originalColumnBack = transformedData_default.GetColumn<VBuffer<ReadOnlyMemory<char>>>(ml, defaultColumnName);
+
+            foreach (var row in originalColumnBack)
+            {
+                foreach (var value in row.Values)
+                    Console.Write($"{value} ");
+                Console.WriteLine("");
+            }
+
+            // DefaultKeys 
+            // radiation galaxy universe duck
+            // space galaxy universe radiation
+            // bus pickup universe radiation
+            // car truck universe radiation
```

Comment:
>universe radiation [](start = 25, length = 18)

this seems to be a bug too.

Reply:
right, will investigate post PR. 

---
In reply to: [228249405](https://github.com/dotnet/machinelearning/pull/1321#discussion_r228249405) [](ancestors = 228249405)